Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+{
+    "appid": "00a4cb00-19a1-46f6-9923-f502a3dd346b",
+    "name": "GitLab",
+    "description": "This app integrates with a GitLab instance to perform various lookups and execute CI/CD pipelines.",
```

Comment:
No period at the end

Reply:
I removed the period in [this commit](https://github.com/peterbertel/phantom-apps/commit/8159a235f9d415edc12c6442f09df319460868e2).